src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/Decider.java

```diff
@@ -41,10 +45,12 @@
 
   private static final Logger LOG = LogManager.getLogger(Decider.class);
   protected final int decisionFrequency; // Measured in terms of number of evaluationIntervalPeriods
+  private final HashMap<Decider, ArrayList<String>> deciderActionPriorityOrder;
 
   public Decider(long evalIntervalSeconds, int decisionFrequency) {
     super(0, evalIntervalSeconds);
     this.decisionFrequency = decisionFrequency;
+    this.deciderActionPriorityOrder = ConfigReader.getDeciderActionPriorityOrder();
```

Comment:
If `ConfigReader` can store the priority hashmap in memory, then each decider doesn't need to store it. Config reader can expose the getter methods

Reply:
Modified in the latest version.